kolibri/content/test/test_contentdirectory.py

```diff
@@ -0,0 +1,119 @@
+from django.core.management import call_command
+from django.test import TestCase
+from mock import patch
+
+from kolibri.utils.conf import OPTIONS
+
+
+class ContentGetDirectoryTestCase(TestCase):
+    """
+    Testcase for the command getdirectory
+    """
+
+    @patch('kolibri.content.management.commands.getdirectory.Command.show')
+    @patch('kolibri.content.management.commands.getdirectory.os.path.exists', return_value=False)
+    def test_get_current_content_dir_dne(self, path_exists_mock, show_mock):
+        with self.assertRaises(SystemExit):
+            call_command('getdirectory')
+            show_mock.assert_not_called()
+
+    @patch('sys.stdout')
+    @patch('kolibri.content.management.commands.getdirectory.os.path.exists', return_value=True)
+    def test_get_current_content_dir_exists(self, path_exists_mock, stdout_mock):
+        call_command('getdirectory')
+        stdout_mock.call_count = 1
+
+
+class ContentSetDirectoryTestCase(TestCase):
+    """
+    Testcase for the command setdirectory <destination>
+    """
+
+    @patch('kolibri.content.management.commands.setdirectory.rmtree')
+    @patch('kolibri.utils.server.get_status', return_value=True)
+    @patch('kolibri.content.management.commands.setdirectory.os.path.exists', return_value=True)
+    def test_migrate_while_kolibri_running(self, path_exists_mock, server_mock, remove_mock):
+        with self.assertRaises(SystemExit):
+            call_command('setdirectory', 'test')
+            remove_mock.assert_not_called()
+
+    @patch('kolibri.content.management.commands.setdirectory.update_options_file')
+    @patch('kolibri.content.management.commands.setdirectory.rmtree')
+    @patch('kolibri.content.management.commands.setdirectory.copy_tree')
+    @patch('kolibri.content.management.commands.setdirectory.input', return_value='yes')
+    @patch('kolibri.content.management.commands.setdirectory.os.listdir')
+    @patch('kolibri.content.management.commands.setdirectory.os.path.exists', return_value=True)
```

Comment:
A lot of patching here :)

Reply:
yes... 😓 will try to remove some to make it simple